### DISCUSSION TITLE
Photo's app repeatedly crashes

### DISCUSSION THREAD

**Discussion by doobes** [elementary/files#2117] [General]
I changed the photo folder that Photo's uses and it starts the process of "Updating the library".

Then the app crashes to desktop. Repeatedly.

Oopies...

**Comment by jeremypw** [comment]
Are you sure this is a Files issue?  It sounds like you are using the Photos app (that is the one that has an "updating library").  Or is a Files window open at the same as Photos and the Files window closes?  If so, is Files showing the same folder as that being updated?

**Comment by jeremypw** [comment]
Closing as uncertain which app is responsible and no further response.
